app/assets/stylesheets/_bourbon-deprecate.scss

```diff
@@ -0,0 +1,19 @@
+@charset "UTF-8";
+
+/// Throws Sass warnings to announce library deprecations. People can disable
+/// these using the `$disable-bourbon-deprecation-warnings` variable.
```

Comment:
There is also something funny to me about the double negative here. What if we were to name the variable `show-bourbon-deprecation-warnings` or `output-bourbon-deprecation-warnings` or something that will get us to a place where we are more direct with `false` turning it off, and `true` turning it on.